docs/docs/build/connectors/data-source/https.md

```diff
@@ -23,20 +25,25 @@ Choose the method that matches your data source requirements.
 
 ### Authenticated Endpoints
 
-If your endpoint requires authentication, create a `https` connector that can test your API key before importing data:
+If your endpoint requires authentication, create a `https` connector that includes your authentication credentials. The connector defines the headers that will be used when accessing HTTPS sources. Here's an example connector configuration:
 
 ```yaml
 type: connector 
 driver: https 
-path: "https://api.endpoint.com/v1" 
 
 headers:
     Authorization: "Bearer {{ .env.connector.https.token }}"
 ```
 
 ### Public URLs
 
-For publicly accessible files, you can simply use the `https` connector to add data from your endpoint or publicly available file.
+For publicly accessible files, you don't need to create a connector at all. Simply specify the URL directly in your source or model configuration. If you need to use a connector (for example, to set up path prefixes), you can create one without headers:
+
+```yaml
+type: connector
+driver: https
+# No headers needed for public endpoints
+```
 
 ## Adding an HTTPS Source
```

Comment:
The section about connectors above doesn't mention the second step: actually providing the HTTP(S) URL. Shouldn't it?

If presenting both the Connector paradigm and the Source paradigm, can we be clear that there are two paradigms to choose from and call out which one is recommended?